Cargo.toml

```diff
@@ -25,7 +25,7 @@ include = [
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "0.19"
+tree-sitter = ">= 0.19, < 0.21"
```

Comment:
This change is already present in the recently merged f0995f0a3705cf4a21a824a3a65f6a610301a0c6